tests/integration/providers/ldap/delta_inbound_sync_tests.py

```diff
@@ -583,7 +581,32 @@ def test_create_fake_user(ldap_connection, user):
     time.sleep(1)
     email = "%s@clouddev.corporate.t-mobile.com" % user["common_name"]
     result = is_user_in_db(email)
+    syncflag_fetched = get_syncmatch_count(user["common_name"])
     assert result is True
+    assert syncflag_fetched is True
+
+
+def get_syncmatch_count(username_provided):
+    """
+    Function that results TRUE or False,
+    If the count of result is greater than 0
+    And, TRUE is returned if all the users that are inserted
+    into the users table have metadata field set to
+    sync_direction : OUTBOUND
+    """
+    with connect_to_db() as db_connection:
+        count_result = (
+            r.table("users")
+            .filter(
+                {
+                    "username": username_provided,
+                    "metadata": {"sync_direction": "INBOUND"},
+                }
+            )
+            .count()
+            .run(db_connection)
+        )
+        return count_result > 0
```

Comment:
with getting the count of all users with a matching username and `sync_direction` set to `INBOUND` won't this comparator return `False` if all matching users are set to `OUTBOUND`? I think either the query or docstring has a typo

Reply:
docstrings has the typo and modifying it, accordingly.